src/frontend/komponenter/Brevmottakere/BrevmottakereListe.tsx

```diff
@@ -48,14 +48,21 @@ export const BrevmottakereListe: FC<Props> = ({
             prevState.filter((mottaker) => mottaker.organisasjonsnummer !== organisasjonsnummer)
         );
     };
+
+    const mapTilMottakerNavn = (mottaker: IBrevmottaker) => {
+        if (mottaker && mottaker.navn) {
+            return mottaker.navn;
+        } else return '';
+    };
+
     return (
         <>
             <Undertittel>Brevmottakere</Undertittel>
             {valgtePersonMottakere.map((mottaker, index) => (
                 <StyledMottakerBoks key={mottaker.navn + index}>
                     <Flexboks>
                         <BodyShort>
-                            {`${mottaker.navn} (${mottaker.mottakerRolle.toLowerCase()})`}
+                            {`${mapTilMottakerNavn(mottaker)} (${mottaker.mottakerRolle.toLowerCase()})`}
```

Comment:
```suggestion
                            {`${mottaker?.navn || ''} (${mottaker.mottakerRolle.toLowerCase()})`}
```

`mapTilMottakerNavn` er unødvendig